test/__mocks__/network.js

```diff
@@ -35,6 +36,7 @@ function getData(url) {
       path.resolve(__dirname, `./data/${directory}/proxy_response.json`)
     );
     const data = JSON.parse(dataFile);
+    const response = data[url];
```

Comment:
How is this `response` variable is being used ?

Reply:
This is what we get as input in `responseHandler`.

Reply:
I don't think the variable `response` is being used anywhere else. Can we remove this line if that is the case ?